dispatcher/docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10-slim
+
+WORKDIR /usr/app/src
+
+COPY ./requirements.txt /usr/app/src/requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY ./templates /usr/app/templates
+COPY ./dispatcher/main.py /usr/app/src/main.py
+
+#USER python
+
+#ENV FOO=BAR
```

Comment:
```suggestion
```
unless it is needed for local dev?

Reply:
USER python should be there for sure, as otherwise it's from the root account and it's bad security - I was only cutting corner for development